.github/workflows/ci.yaml

```
@@ -0,0 +1,18 @@
+---
+
+on:
+  - pull_request
+  - push
+
+name: 'Run shellcheck'
```

Technically this could be
```suggestion
name: CI
```

Then the naming ends up being `CI / Shellcheck`